### PR TITLE
feat: home page made responsive.

### PR DESCRIPTION
**Pull request by krishnakant01**
Hey I've added responsiveness to home page

### CODE REVIEW

File: components/navigation/navigation.css [hunk 1]
```diff
@@ -164,16 +164,22 @@
...
 
     body {
-        overflow: hidden;
+        overflow-x: hidden;
         transition: 0.35s ease-out;
     }
 
     .mobile-nav-toggle {
+        position: sticky;
+
```
Comment (Kothalkarpriya) [inline components/navigation/navigation.css:173]: remove unnecessary spacings

File: docSite-components/docsite-sidebar.css [hunk 2]
```diff
@@ -1,4 +1,4 @@
-.sidebar-header {
+/* .sidebar-header {
     margin: 2rem 2rem 0rem 3rem;
```
Comment (Kothalkarpriya) [inline docSite-components/docsite-sidebar.css:2]: remove comments which are not in use

**Review by Kothalkarpriya — CHANGES_REQUESTED** [review]
COde looks good, just remove unnecessary spacing and comments

**Review by Nagashree01 — APPROVED** [review]
Looks good to me. Just remove unnecessary spaces

**Pull request merged**